components/Aboutus/WhyChooseUs.js

```diff
@@ -0,0 +1,11 @@
+export default function WhyChooseUs({ title, subtitle, Logo }) {
+  return (
+    <div className="bg-white flex flex-row lg:p-4 md:p-2 rounded-xl place-content-center">
```

Comment:
Add `shadow-md` class to it and don't center align the elements in the card. Also reduce the width, they are too big